app/javascript/packs/contentDisplayPolicy.js

```diff
@@ -1,10 +1,18 @@
 import { hideBlockedContent } from '../contentDisplayPolicy/hideBlockedContent';
 import { initHiddenComments } from '../contentDisplayPolicy/initHiddenComments';
+import { embedGists } from '../utilities/gist';
```

Comment:
Refactor (blocking): Although this works, we want to avoid loading JavaScript when it's not necessary, so even though there wouldn't be much in it, create another pack that only gets loaded on the notifications page.

Reply:
@nickytonline aha ^^. Love that app. I'll make the new pack.